test/Ocelot.UnitTests/Ocelot.UnitTests.csproj

```diff
@@ -75,7 +75,7 @@
     <PackageReference Include="CacheManager.Core" Version="2.0.0-beta-1629" />
     <PackageReference Include="CacheManager.Microsoft.Extensions.Configuration" Version="2.0.0-beta-1629" />
     <PackageReference Include="CacheManager.Microsoft.Extensions.Logging" Version="2.0.0-beta-1629" />
-    <PackageReference Include="Polly" Version="8.3.1" />
+    <PackageReference Include="Polly" Version="8.4.0" />
```

Comment:
I'm not sure but the reference should be auto-restored because of ref to `Ocelot`, right?